engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -83,6 +99,17 @@ public String get() {
         final UIList<GameInfo> gameList = find("gameList", UIList.class);
 
         refreshList(gameList);
+
+        gameList.subscribeSelection((widget, item) -> {
+            find("load", UIButton.class).setEnabled(item != null);
+            find("delete", UIButton.class).setEnabled(item != null);
+//            find("details", UIButton.class).setEnabled(item != null);
```

Comment:
This is IMHO OK to leave commented out in this PR if you're about to activate it next PR. In that sort of case I'd leave a `//TODO: reason` at the end of the line.  Not needed if you're just about ready to enable it next PR already

Reply:
Too lazy to remove it:) Basically, I'll create my next PR based on this branch, and I'll take care about this line.